package.json

```diff
@@ -8,11 +8,12 @@
   },
   "workspaces": [
     "plugins/*",
-    "support/*"
+    "support/*",
+    "utils/*"
   ],
   "scripts": {
     "clean": "find ./ '(' -name 'node_modules' -o -name 'dist' -o -name '.turbo' ')' -type d -exec rm -rf {} +",
-    "test": "turbo test",
+    "test": "turbo test --force",
```

Comment:
Adding the `--force` flag disables cache reads allowing the full test suite to be run (and pass). Without this flag the `automap` plugin's tests will fail when run from the root.